packages/mobile/src/i18n/i18nSlice.ts

```diff
@@ -0,0 +1,60 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit'
+import { Actions as AppActions, UpdateConfigValuesAction } from 'src/app/actions'
+import { REMOTE_CONFIG_VALUES_DEFAULTS } from 'src/firebase/remoteConfigValuesDefaults'
+import { getRehydratePayload, REHYDRATE, RehydrateAction } from 'src/redux/persist-helper'
+
+export interface State {
+  language: string | null
+  allowOtaTranslations: boolean
+  otaTranslationsLastUpdate: number
+  otaTranslationsAppVersion: string
+  otaTranslationsLanguage: string
+}
+
+const initialState: State = {
+  language: null,
+  allowOtaTranslations: REMOTE_CONFIG_VALUES_DEFAULTS.allowOtaTranslations,
+  otaTranslationsLastUpdate: 0,
+  otaTranslationsAppVersion: '0',
+  otaTranslationsLanguage: '',
+}
+
+export interface OtaTranslationsUpdatedAction {
```

Comment:
Looks like this doesn't need to be exported.

Reply:
ah yes you're right! i've removed 🙈